error_reporting/snippets/report_exception_test.py

```diff
@@ -18,7 +18,7 @@
 
 
 def test_error_sends():
-    report_exception.simulate_error()
+    report_exception.report_exception()
```

Comment:
suggestion: This test doesn't seem to `test` anything besides running the code. Probably not in scope of this change, but could we log an issue to extend this test to check if error_reporting actually worked?